lib/src/utils/getBandwidthBps.ts

```diff
@@ -0,0 +1,13 @@
+import type { ResourceTiming } from '../request/ResourceTiming';
+
+/**
+ * Converts a ResourceTiming sample to bandwidth in bits per second (bps).
+ *
+ * @param sample - A ResourceTiming sample
+ * @returns
+ */
+export const getBandwidthBps = (sample: ResourceTiming): number => {
+	const durationSeconds = sample.duration / 1000;
+	const bandwidthBps = sample.encodedBodySize * 8 / durationSeconds;
+	return bandwidthBps;
+};
```

Comment:
Minor change: Let's use the same function declaration syntax as the other utility functions:
```suggestion
export function getBandwidthBps(sample: ResourceTiming): number {
	const durationSeconds = sample.duration / 1000;
	const bandwidthBps = sample.encodedBodySize * 8 / durationSeconds;
	return bandwidthBps;
}
```